jenkins/runlocal

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash -e
 
 # Build the pbench-server RPM locally, then build the containers locally, and
-# then run the fuctional tests against the locally built CI container image.
+# then run the functional tests against the locally built CI container image.
 export PB_SERVER_IMAGE_NAME=pbench-server
 
 # We use the current user name as a the tag to avoid any conflict with what the
 # CI environment does.
 export PB_SERVER_IMAGE_TAG=${USER}
 
-# We use the image pull policy of Never here to ensure our locally build image
+# We use the image pull policy of Never here to ensure our locally built image
 # is used by the pod.
 export PB_SERVER_IMAGE_PULL_POLICY=Never
 
-# We explicit set WORKSPACE_TMP so that RPM build (which uses a container image)
-# will store the resulting RPM in that work space temporary directory so we can
-# reference it when building the container images.
+# We explicit set WORKSPACE_TMP so that RPM builds (which uses a container
+# image) # will store the resulting RPM in that work space temporary directory
```

Comment:
"We explicitly set WORKSPACE_TMP so that RPM builds will store the resulting RPMs" ...

I don't see how the "(which uses a container image)" is particularly relevant (nor even true?) here, and the following "#" is left over from a re-wrap.

Reply:
Fixed.